README.md

```diff
@@ -277,7 +277,7 @@ bazel-selenium/external/bundle/bin/rdbg -A
 
 If you want to use RubyMine for development, a bit of extra configuration is necessary to let the IDE know about Bazel toolchain and artifacts:
 
-1. Run `bazel build @bundle//:bundle //rb:selenium-devtools //rb:selenium-webdriver` before configuring IDE.
+1. Run `bundle exec rake update` as necessary to update generated artifacts.
```

Comment:
I still think you don't need to include rake in the dev dependencies, but feel free to merge if you don't want to both bother updating.

Reply:
Also, this should be run from `rb/` directory, not the root.

Reply:
Hmm, this could go below step 2 where we open `rb/` as main project directory. And yeah, I guess I kind of miss the old `./go` wrapper. 😂